### PR TITLE
Use py3's f-strings instead of s.format(**locals())

### PR DESCRIPTION
**Pull request by mistercrunch**
In light of the bug reported here
https://github.com/apache/incubator-superset/issues/6347, I greped and replaced all
instances of `.format(**locals())` using py3's f-strings

### CODE REVIEW

**Comment by mistercrunch** [comment]
BTW we could do a wider sweep and replace all `.format()` with f-strings.

File: tox.ini [hunk 1]
```diff
@@ -20,6 +20,8 @@ ignore =
     FI51
     FI53
     FI54
+    W504
```
Comment (john-bodley) [inline tox.ini]: Why are we excluding more warning types?

Comment (mistercrunch) [reply tox.ini]: New bad warning came up after bumping flake8 / pycodestyle, rolling back this bumping.

Comment (mistercrunch) [reply tox.ini]: Trying de-bump flake8 / pycodestyle

**Review by john-bodley — APPROVED** [review]

**Pull request merged**